### DISCUSSION TITLE
'body: ReadableStream' prevents to send data to backend

### DISCUSSION THREAD

**Discussion by IkeCairns** [prisma/prisma#22156] [Q&A]
### Question

Hi, I am building an app by NextJS 14. I managed to install Prisma. However, when I tried to send the client input data by onSubmit event, the data did not show up on Prisma Studio. The console just returned 'body: ReadableStream { locked: false }' on response. This response should have been the client input data. I would like to reflect the data to Prisma Studio.

Github: https://github.com/IkeCairns/ContactPage-Public

### How to reproduce (optional)

On Terminal
1. sudo docker compose up --build -d
2. npx prisma generate
3. sudo docker exec -it <myblogname> sh
4. <after /app #> npx prisma studio
5. pnpm dev

On the client browser
1. Input some form data and press submit button
2. <after /app #> npx prisma db push
3. <after /app #> npx prisma studio

Nothing is showed up on the browser 'localhost://5555'

### Expected behavior (optional)

The client input data should be transferred to the backend, and it also should reflect the Prisma Studio.

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

model Post {
  id         Int      @id @default(autoincrement())
  name       String
  email      String   @unique
  message    String
  created_at DateTime @default(now())
}
```
```ts
//  route.ts

import { NextResponse, NextRequest } from "next/server";
import prisma from "src/lib/prisma";

// export async function GET(){
//   const posts = await prisma.post.findMany();
//   return NextResponse.json(posts)
// }

export async function POST(req: NextRequest) {
  const { name, email, message } = await req.json()
  const result = prisma.post.create({
    data: {
      name,
      email,
      message
    }
  })
  return NextResponse.json(result)
}
```
```tsx
// form.tsx

"use client"

import { useState } from "react"
import { useRouter } from "next/navigation"

export const PostForm: React.FC = () => {
  const [name, setName] = useState("");
  const [email, setEmail] = useState("");
  const [message, setMessage] = useState("");
  const router = useRouter();

  const onSubmit = async (e: React.SyntheticEvent) => {
    e.preventDefault()
    try {
      const body = { name, email, message }
      const result =await fetch("/api/post", {
        method: "POST",
        headers: { "Content-type": "application/json" },
        body: JSON.stringify({ body }),
      });

      console.log(result);
      router.push("/welcome");
      return result;

    } catch (error) {
      console.error(`Error submitting the form`, error);
    }
  }
  
  return (
    <>
      <form onSubmit={onSubmit}>
        <div>
          <label htmlFor="name">Name</label>
          <input
            value={name}
            onChange={ e => setName(e.target.value)}
            type="text"
            placeholder="Your Name"
          />
        </div>
        <div>
          <label htmlFor="email">Email</label>
          <input
            value={email}
            onChange={ e => setEmail(e.target.value)}
            type="email"
            placeholder="abc@example.comf"/>
        </div>
        <div>
          <label htmlFor="inquiry">Message</label>
          <textarea 
            value={message}
            onChange={ e => setMessage(e.target.value)}
            placeholder="Message">
          </textarea>
        </div>
        <button type="submit">Submit</button>
      </form>
    </>
   )
  }
  ```
```
- OS: Ubuntu 22.04.3 LTS
- Database: PostgreSQL
- Node.js version: v20.7.0
- Prisma version: 5.6.0
```

**Comment by IkeCairns** [comment]
Nobody replies to this post, and now I have got other concerns so that I close this discussion.

**Reply by ludralph** [reply]
Hi @IkeCairns 👋 

Apologies for the late reply. I did take a look at this but kept running into an error after issuing the command `sudo docker compose up --build -d` as can be seen from the screenshot below:

![image](https://github.com/prisma/prisma/assets/13855608/e62faeaf-4a44-4d57-8dad-ad5658821a5a)

Hopefully, I will work out the error encountered and provide updates to you on this.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
